test/core/crypto/murmur_test.cpp

```diff
@@ -0,0 +1,18 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "crypto/murmur/murmur.hpp"
+
+#include <gtest/gtest.h>
+#include "testutil/literals.hpp"
+
+/**
+ * @given Bytes
+ * @when Hash
+ * @then Equals to pregenerated hash
+ */
+TEST(Murmur, GoCompatibility) {
```

Comment:
pls add more hash tests

Reply:
and make them parameterized, I mean, several values in a row processed by one test

Reply:
as an example:
https://github.com/soramitsu/kagome/blob/bd9632482aa1c4bb773a3ddf59766e36dca7eb79/test/core/scale/scale_compact_test.cpp#L40